releases/release-1.31/release-notes/maps/pr-123905-map.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 123905
+releasenote:
+  text: |-
+    Services implemented a field selector for the ClusterIP and Type fields.
+    Kubelet uses the fieldselector on Services to avoid watching for Headless Services and reduce the memory consumption.
```

Comment:
```suggestion
    Kubelet uses the fieldselector on Services to avoid watching for Headless Services and reduce the memory consumption.
```
```suggestion
    The Kubelet uses this field selector to avoid monitoring Headless Services, which helps reduce memory consumption.
```